QEfficient/utils/logging_utils.py

```diff
@@ -56,3 +58,13 @@ def create_logger() -> logging.Logger:
 
 # Define the logger object that can be used for logging purposes throughout the module.
 logger = create_logger()
+
+
+def tabulate_measurements(fields, file):
+    if not os.path.exists(file):
+        with open(file, "w") as csvfile:
+            csvwriter = csv.writer(csvfile)
+            csvwriter.writerow(list(fields.keys()))
+    with open(file, "a", newline="") as csvfile:
+        csvwriter = csv.writer(csvfile)
+        csvwriter.writerow(list(fields.values()))
```

Comment:
This will keep on writing to the same CSV, as long as nobody deletes it.
Should we always create new CSV with timestamp and let user decide how to combine those CSVs.

If single CSV keeps updating, it might have too much data and it might be hard to distinguish which data is relevant.

Reply:
I believe it is needed to accumulate the measurements for the same model in the same CSV file via various runs. Later it is easy for the user to sort through this file and compare against the previous measurements.